app-rails/app/previews/breadcrumbs_preview.rb

```diff
@@ -0,0 +1,9 @@
+class BreadcrumbsPreview < Lookbook::Preview
+  layout "component_preview"
+
+  def default
+    render template: "application/_breadcrumbs", locals: { crumbs: [
```

Comment:
`render partial: "application/breadcrumbs"` not supported by lookbook so use `render template:` and remember to prefix with _ and set layout to "component_preview" to get rid of application layout